addons/nightvision/XEH_PREP.hpp

```diff
@@ -1,7 +1,13 @@
+LOG("prep");
```

Comment:
Remove.

Reply:
I really like these for verify the recompile took effect.
`LOG` macro is disabled normally
I'll remove for now but maybe we can add them to all modules